builder/builder-next/builder.go

```diff
@@ -385,7 +402,7 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 		if err != nil {
 			return err
 		}
-		if exporterName != "moby" {
+		if exporterName != mobyexporter.Moby {
```

Comment:
```suggestion
		if exporterName != "moby" && exporterName != client.ExporterImage {
```

We also want the image ID to be emitted when `image` exporter is used.
Fork PR: https://github.com/rumpl/moby/pull/125

Reply:
Oh! Hmm.. so there _is_ a list of exporter `consts`, but `"moby" !== "docker"` (for exporter), correct? Or are they the same?; https://github.com/moby/buildkit/blob/4b220de5058abfd01ff619c9d2ff6b09a049bea0/client/exporters.go#L4-L8

```go
const (
	ExporterImage  = "image"
	ExporterLocal  = "local"
	ExporterTar    = "tar"
	ExporterOCI    = "oci"
	ExporterDocker = "docker"
)
```

Reply:
According to [ExporterDocker](https://github.com/moby/buildkit#docker-tarball) `docker` basically exports a tar with Docker image format (tarred blobs + manifest.json).

Reply:
There is a list in buildkit indeed, that's the list of exporters defined by buildkit, the `moby` exporter is defined in moby's code, not in buildkit